addons/binding/org.openhab.binding.neeo/src/main/java/org/openhab/binding/neeo/NeeoUtil.java

```diff
@@ -0,0 +1,98 @@
+/**
```

Comment:
Do you use these methods from more than one class? Can you move them to the class to which they belong.
I kind of follow the thinking from the discussion [here](https://stackoverflow.com/questions/3340032/utility-classes-are-evil)

Reply:
Yes - they are used in many classes and are really generic (crosscut throughout the binding).  If we were allowed to cross binding lines, I'd have this class at an even higher level since every binding of mine defines it with the same exact signature